### PR TITLE
Fixing the missing sample_rate argument in mixin calling in Sortformer model file

### PR DESCRIPTION
**Pull request by tango4j**
# What does this PR do ?

This PR fixes the missing sample_rate argument in mixin calling in Sortformer model file

**Collection**: 

ASR/speaker_tasks

# Usage

- You can potentially add a usage example below

```python
from nemo.collections.asr.models import SortformerEncLabelModel

# load model from Hugging Face model card directly (You need a Hugging Face token)
diar_model = SortformerEncLabelModel.from_pretrained("nvidia/diar_streaming_sortformer_4spk-v2")

# If you have a downloaded model in "/path/to/diar_streaming_sortformer_4spk-v2.nemo", load model from a downloaded file
diar_model = SortformerEncLabelModel.restore_from(restore_path="/path/to/diar_streaming_sortformer_4spk-v2.nemo", map_location='cuda', strict=False)

# switch to inference mode
diar_model.eval()

predicted_segments = diar_model.diarize(audio=audio_input, batch_size=1, sample_rate=16000)

# This PR makes sure there should be "sample_rate" for numpy input
```

# GitHub Actions CI

The Jenkins CI system has been replaced by GitHub Actions self-hosted runners.

The GitHub Actions CI will run automatically when the "Run CICD" label is added to the PR.
To re-run CI remove and add the label again.
To run CI on an untrusted fork, a NeMo user with write access must first click "Approve and run".

# Before your PR is "Ready for review"

**Pre checks**:

- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [x] Did you write any new necessary tests?
- [x] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:

- [ ] New Feature
- [x] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in NeMo ASR

# Additional Information

User (ooobo) pointed out this issue in [PR15184](https://github.com/NVIDIA-NeMo/NeMo/pull/15184)

### CODE REVIEW

**Review by KunalDhawan — APPROVED** [review]
LGTM!
